proposed/http-message.md

```diff
@@ -542,7 +538,7 @@ interface RequestInterface extends MessageInterface
      * @see MessageInterface::getHeader()
      * @see UriInterface::getHost()
      * @param string $name Case-insensitive header field name.
-     * @return string
+     * @return array|null
```

Comment:
No; the idea is that it returns a consistent type (array), so you can always use the return value for iteration.